content/docs/integrations/okta/index.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Configuring Okta Team / User Provider
+publishedDate: '2022-08-02T17:00:00.0Z'
+description: How to configure Roadie to collect users from Okta.
+
+humanName: Okta
+integrationType: OSS plugin
+---
+
+## Introduction
+
+In order to use the Okta organization entity plugin, Roadie needs an API token, and the organization url to communicate with Okta APIs.
+
+This page describes how to create and set up the API token.
+
+## Prerequisites
+You will need to [configure an Okta API token](https://developer.okta.com/docs/guides/create-an-api-token/) for use by Backstage.
```

Comment:
```suggestion
Obtain an [Okta API token](https://developer.okta.com/docs/guides/create-an-api-token/) to use with Roadie Backstage. 
```

Also, I suggest you change this from "Prerequisites" to "Step 1"

Reply:
done. thanks